### ISSUE TITLE
Offline Chat Not Downloading

### ISSUE THREAD

**Issue by KatakanaKatana** [irsat000/CAI-Tools#98]
Hey there, 

I have a pretty long running chat that I've been working on with a character. I typically download the offline chat every now and then just to be safe. 

This time, after running the 'Fetch' and getting the "Ready" message, I don't get anything after pressing the Offline Chat button for download. I wait quite a while, expecting to take some time since it's a long chat, but no dice this time. 

Any solutions or work arounds?

I'm running on Opera GX if that matters. I've never had this problem before.

**Comment by irsat000** [comment]
Start Fetch button is for Offline History. Offline Chat is the current chat and its fetch is automatic. You should see "Loading" next to it, and Ready when it's finished under "Conversation" title.

Since a part of it has no error message showing, try this one;
[CAI-Tools-ChatDownloadTest 1.zip](https://github.com/user-attachments/files/18281261/CAI-Tools-ChatDownloadTest.1.zip)

[How to](https://help.opera.com/en/extensions/basics/#:~:text=Go%20to%20the%20browser%20address,Select%20your%20extension's%20directory) for Opera. Don't forget to extract the folder.

**Comment by KatakanaKatana** [comment]
> Start Fetch button is for Offline History. Offline Chat is the current chat and its fetch is automatic. You should see "Loading" next to it, and Ready when it's finished under "Conversation" title.
> 
> Since a part of it has no error message showing, try this one; [CAI-Tools-ChatDownloadTest 1.zip](https://github.com/user-attachments/files/18281261/CAI-Tools-ChatDownloadTest.1.zip)
> 
> [How to](https://help.opera.com/en/extensions/basics/#:~:text=Go%20to%20the%20browser%20address,Select%20your%20extension's%20directory) for Opera. Don't forget to extract the folder.

Hey there, sorry to bother about this again. I don't know what changed, if there was an update but chat export worked a few days ago and it's back to not working again.

I'm not very tech savvy, so could you possibly provide further assistance in layman's terms? I see several json files and am not sure which I'm supposed to upload. A concern I have in testing in uploading this unpackaged folder is losing all of my work in the Memory Manager for some of my chats/

**Comment by irsat000** [comment]
It's probably because of the plus.character.ai bug that rejects character.ai requests.

Right click on page, inspect, go to console tab. What do you see there, could you copy paste?

**Comment by KatakanaKatana** [comment]
![Image](https://github.com/user-attachments/assets/09cca961-ab63-4294-abb9-2a6ad90db47c)
![Image](https://github.com/user-attachments/assets/eb8fb32d-fdc0-41f0-ad78-1f65c50552df)

There seem to be quite a few errors.. not sure what any of it means though, unfortunately.

P.s. thanks for walking me through this!

**Comment by irsat000** [comment]
Yeah it's a website bug. Try disabling vpn if you have it. Or try changing location of it.

Since the entire website is working when this bug happens according to someone, I will try to see how I could go around it.

**Comment by KatakanaKatana** [comment]
Ahh! Disabling the VPN seemed to work! Thank you!

**Comment by irsat000** [comment]
I will try using vpn to get the same result. Then I can maybe fix it.
